engine/realtimelogger.cpp

```diff
@@ -0,0 +1,66 @@
+// spdlog must be included before fmt. Since spdlog uses fmt, it will cause errors otherwise
+#include <spdlog/spdlog.h>
```

Comment:
Some funkiness because spdlog uses fmt under the hood. Not sure if there is a good way around it.